docs/exercise004.md

```diff
@@ -0,0 +1,18 @@
+# Exercise 004
+
+This exercise gives you the opportunity to explore classes, Date/Time and unit tests in Java.
+
+### Instructions
+Given a moment, your job is to write a program to find the moment that would be after a gigasecond has passed.
```

Comment:
Hey @kimdiep 
I feel like this one could be explained a little better. Maybe a subtle change to

"Given a moment in time (represented as a date/time value), your job is to....."

Reply:
Thanks @eggsy84 - agreed :) Will make the change now :)

Reply:
Hi @eggsy84 - that change has been implemented now :) If you could re-review, that would be super duper :)

Reply:
Hey @kimdiep 
Yep that looks good to me. More descriptive on the moment. 👍